src/OMSimulatorLib/Flags.h

```diff
@@ -158,6 +165,7 @@ namespace oms
       {"--solver", "", "Specifies the integration method (euler, [cvode])", re_default, Flags::Solver, false},
       {"--solverStats", "", "Adds solver stats to the result file, e.g. step size; not supported for all solvers (true, [false])", re_bool, Flags::SolverStats, false},
       {"--startTime", "-s", "Specifies the start time", re_double, Flags::StartTime, false},
+      {"--stepSize", "", "Set step size", re_stepSize, Flags::StepSize, false},
```

Comment:
Add a hint on how to use `--stepSize` with minimum and maximum step size.

Reply:
done.